packaging/bin/engine-backup.sh.in

```diff
@@ -173,7 +173,8 @@ BACKUP_PATHS="/etc/ovirt-engine
 # Add /var/lib/ovirt-engine except a few
 VAR_LIB_OVIRT_ENGINE_EXCLUSIONS="/var/lib/ovirt-engine/backups
 /var/lib/ovirt-engine/jboss_runtime
-/var/lib/ovirt-engine/ansible-runner-service.log"
+/var/lib/ovirt-engine/ansible-runner-service.log
+/var/lib/ovirt-engine/ansible-runner"
```

Comment:
what about the /var/lib/ovirt-engine/.ansible dir?

Reply:
please exclude it as well

Reply:
ansible creates $HOME/.ansible automatically, we don't set there anything, so it should be safe to omit